Void Linux uses -D_FORTIFY_SOURCE=2. Remove the flag to not interfere (just in case).

--- a/inc/MyBuilder.pm	2019-04-29 17:02:41.000000000 +0200
+++ b/inc/MyBuilder.pm	2019-11-24 08:35:16.561126918 +0100
@@ -208,7 +208,7 @@
         $object =~ s/\.c/.o/;
         next if $self->up_to_date($file, $object);
         $cbuilder->compile(object_file  => $object,
-                           extra_compiler_flags=>["-D_FORTIFY_SOURCE=1",@EXTRA_FLAGS],
+                           extra_compiler_flags=>[@EXTRA_FLAGS],
                            source       => $file,
                            include_dirs => ["btparse/src"]);
     }

